src/config/containers-and-vms/lxc.md

```diff
@@ -114,6 +119,40 @@ access to other containers. To isolate containers from each other, alter the
 create each container. Trying to fix permissions on a container created with the
 wrong map is possible, but inconvenient.
 
+## Incus
+
+Incus provides an alternative interface to LXC's `lxc-*` utilities. However, it
+does not require the configuration described in [the previous section](#lxc).
+
+In `/etc/rc.conf`, set the `CGROUP_MODE` variable to `unified`. Install the
+`incus` package, and [enable](../services/index.md#enabling-services) the
+`incus` service.
```

Comment:
maybe we could mention looking in README.voidlinux for optional dependencies? like `Some parts of Incus require optional dependencies, see [README.voidlinux](../package-documentation/index.md).` (link might not be 100% right)

Reply:
Sounds good